requirements.txt

```diff
@@ -1,6 +1,16 @@
+aioredis==1.3.1
 asgiref==3.5.2
 async-timeout==4.0.2
 attrs==22.1.0
+autobahn==22.7.1
+Automat==20.2.0
+backports.zoneinfo;python_version<"3.9"
```

Comment:
Can this be removed? Are we using "backports" anywhere?

Reply:
No description?